core/src/test/java/org/apache/iceberg/rest/responses/TestListNamespacesResponse.java

```diff
@@ -83,6 +83,23 @@ public void testBuilderDoesNotCreateInvalidObjects() {
         .hasMessage("Invalid namespace: null");
   }
 
+  @Test
+  public void testWithNullPaginationToken() {
+    ListNamespacesResponse response =
+        ListNamespacesResponse.builder().addAll(NAMESPACES).nextPageToken(null).build();
+    Assertions.assertThat(response.nextPageToken()).isNull();
+    Assertions.assertThat(response.namespaces()).isEqualTo(NAMESPACES);
+  }
+
+  @Test
+  public void testWithPaginationToken() {
```

Comment:
this should also do a JSON round-trip check similar to `testRoundTripSerDe()`

Reply:
will add this